commands/operator-sdk/cmd/scorecard/olm_tests.go

```diff
@@ -238,95 +238,89 @@ func getUsedResources() ([]schema.GroupVersionKind, error) {
 }
 
 // annotationsContainExamples makes sure that the CSVs list at least 1 example for the CR
-func annotationsContainExamples(csv *olmapiv1alpha1.ClusterServiceVersion) {
-	test := scorecardTest{testType: olmIntegration, name: "CRs have at least 1 example", maximumPoints: 1}
-	if csv.Annotations != nil && csv.Annotations["alm-examples"] != "" {
-		test.earnedPoints = 1
+func (t *AnnotationsContainExamplesTest) Run(ctx context.Context) *TestResult {
+	res := &TestResult{Test: t}
+	if t.CSV.Annotations != nil && t.CSV.Annotations["alm-examples"] != "" {
+		res.EarnedPoints = 1
 	}
-	scTests = append(scTests, test)
-	if test.earnedPoints == 0 {
-		scSuggestions = append(scSuggestions, "Add an alm-examples annotation to your CSV to pass the "+test.name+" test")
+	if res.EarnedPoints == 0 {
+		res.Suggestions = append(res.Suggestions, "Add an alm-examples annotation to your CSV to pass the "+t.GetName()+" test")
```

Comment:
Just to be consistent with the way we format the suggestion strings in other tests.
```suggestion
		res.Suggestions = append(res.Suggestions, fmt.Sprintf("Add an alm-examples annotation to your CSV to pass the %s test", t.GetName()))
```